content/ru/_solutions/abrom_bishkek_fully_automated_house.md

```
@@ -0,0 +1,182 @@
+
+И этот случай не единичный. Умный дом не стоит воспринимать как законченное решение. У заказчика будут появляться новые устройства, и рано или поздно встанет вопрос их интеграции. Возможно, все больше интеграторов будут предоставлять сервисные услуги по добавлению устройств в существующий умный дом. И здесь появится новый рынок.
+
+Этот аспект стоит учитывать при выборе экосистемы умного дома. Возможность расширения функций и добавления устройств в будущем становится все более важной. Поэтому дом должен быть не только умным, но также гибким и расширяемым.
+
+В ассортименте компании Wiren Board есть устройства для решения как простых, так и сложных задач автоматизации и диспетчеризации: программируемые логические контроллеры, датчики климата, счетчики электроэнергии, сетевые карты для холодильников, мощные релейные модули и т.д. Все это позволяет подстроиться под любые требования заказчика и использовать оборудование Wiren Board в его экосистеме. При этом обеспечивается возможность расширения экосистемы в будущем.
+
+Компания Wiren Board даёт на всё оборудование двухлетнюю гарантию с заменой оборудования без предварительной отправки в сервисный центр, а инженеры технической поддержки готовы помочь в решении как типовых, так и нестандартных задач.
+
+## Дополнительные материалы
+
+- [Умный дом босса: что бывает, когда привыкаешь к Алисе](https://habr.com/ru/companies/wirenboard/articles/881962/)
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: asterisk; Actual: dash

(MD004, ul-style)